app/backend/wells/management/commands/export.py

```diff
@@ -195,27 +196,33 @@ def generate_files(self, zip_filename, spreadsheet_filename):
  lhc.lithology_hardness_code = lithology_description.lithology_hardness_code
  left join lithology_colour_code as lcc on
  lcc.lithology_colour_code = lithology_description.lithology_colour_code
- order by well_tag_number""")
+ left join well on well.well_tag_number = lithology_description.well_tag_number
```

Comment:
I think this should be an inner join on the well table. We're not interested in lithology records for unpublished wells.